ServerlessOperations/src/Datahub.Functions/CreateGraphUser.cs

```diff
@@ -65,6 +65,37 @@ public async Task<IActionResult> RunAsync(
             return new BadRequestResult();
         }
     }
+    
+    [Function("AddUserToGroup")]
+    public async Task<IActionResult> AddUserToGroup(
```

Comment:
going to approve this PR since this one is important but could we add a unit test to this method?